examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -35,16 +41,16 @@ const ᶠinterp = Operators.InterpolateC2F(
     top = Operators.Extrapolate(),
 )
 const ᶜdivᵥ = Operators.DivergenceF2C(
-    top = Operators.SetValue(Geometry.Contravariant3Vector(FT(0))),
-    bottom = Operators.SetValue(Geometry.Contravariant3Vector(FT(0))),
+    top = Operators.SetValue(CT3(FT(0))),
```

Comment:
Is `FT` just a global constant defined somewhere?

Reply:
Yeah it's defined in the driver: https://github.com/CliMA/ClimaCore.jl/blob/930a892a89f3e826afd4e30a9c91967f7d7b250c/examples/hybrid/driver.jl#L55